packages/lab/src/divider/Divider.css

```diff
@@ -0,0 +1,21 @@
+.saltDivider-horizontal {
+  border-bottom: 1px solid;
```

Comment:
Is there a reason we're not using background color? Rather than border

Reply:
yes, the tokens are made in consideration to border and also using border, its easy to achieve the vertical divider variant.